src/components/ModelizerModelView.vue

```diff
@@ -141,14 +161,64 @@ async function updatePluginsAndTemplates() {
       templates.value = response;
     })
     .catch(() => {
-      Notify.create({
-        type: 'negative',
-        message: t('errors.templates.getData'),
-        html: true,
-      });
+      notifyError();
     });
 }
 
+/**
+ * Required form compatibility with the HTML drag and drop API.
+ *
+ * @param {DragEvent} event - The drag event.
+ */
+function dragOverHandler(event) {
+  event.preventDefault();
+}
+
+/**
+ * Instantiate from a dragged component definition or template.
+ *
+ * @param {DragEvent} event - The drag event.
+ */
+async function dropHandler(event) {
+  event.preventDefault();
+  const dropData = JSON.parse(event.dataTransfer.getData('text/plain'));
+  const activePlugin = getPluginByName(dropData.pluginName);
+  const modelFolder = process.env.MODELS_DEFAULT_FOLDER
+    ? `${process.env.MODELS_DEFAULT_FOLDER}/${route.query.path}`
+    : `${route.query.path}`;
+
+  let files;
+  if (dropData.isTemplate) {
+    const activeTemplate = templates.value.find(
+      (template) => template.plugin === dropData.pluginName
+        && template.key === dropData.definitionType,
+    );
+
+    if (!activeTemplate) {
```

Comment:
I think this is useless, because when you initiate the drag and drop feature, you have the wanted plugin and the template.
So I think it's uneccesary check